spec/features/application_draft_edit_spec.rb

```diff
@@ -0,0 +1,90 @@
+require 'rails_helper'
+
+describe 'editing application draft' do
+  let!(:draft) { create :application_draft }
+  let!(:top_question) do
+    create :question, number: 1,
+                      application_draft: draft
+  end
+  let!(:middle_question) do
+    create :question, number: 2,
+                      application_draft: draft
+  end
+  let!(:bottom_question) do
+    create :question, number: 3,
+                      application_draft: draft
+  end
+  before :each do
+    when_current_user_is :staff, integration: true
+    visit edit_draft_url(draft)
+  end
+  context 'moving questions' do
+    it 'contains button to move middle question up' do
```

Comment:
Isn't this more of a view spec than an integration spec?  I picture view specs as being "this button exists and this field exists" while integration tests say "clicking this button does this thing."  Am I incorrect in this?